jsonrpc/debug_endpoint.go

```diff
@@ -78,6 +78,10 @@ type debugBlockchainStore interface {
 	// TraceBlock traces all transactions in the given block
 	TraceBlock(*types.Block, tracer.Tracer) ([]interface{}, error)
 
+	// IntermediateRoots executes a block (bad- or canon- or side-), and returns a list
```

Comment:
Sorry for my ignorance, but what does this part mean "(bad- or canon- or side-)"?

Reply:
Copy-paste error, those are block types that don't fit in the blade